app.ts

```diff
@@ -667,7 +667,7 @@ async function tallyVotes(game: InProgressGame) {
 
 async function voteSuccess(game: InProgressGame) {
   // Check if the game is over due to Dillon being promoted
-  if (checkGameOver(game, game.step)) {
+  if (await checkGameOver(game, game.step)) {
```

Comment:
uh how did this ever work?

Reply:
These commits were from 2 months ago and have been deployed before I undeployed them today, so it wasn't broken this whole time.